src/scrabBag.js

```diff
@@ -0,0 +1,21 @@
+export default function scrabBag(theInput, tileDistro = {}) {
+  const inputArray = []
+
+  if (tileDistro === null) {
+    return 'cannot check null'
+  }
+  if (tileDistro.constructor !== Object || tileDistro.constructor === Array) {
+    return 'please use an object to create the distribution of letter tiles'
+  }
+  for (i = 0; i < theInput.length; i++) {
+    inputArray[i] = theInput.charAt([i])
+  }
+  for (k = 0; k < inputArray.length; k++) {
```

Comment:
Whenever I see the pattern:
```js
// Declare temporary collection
// Iterate over collection
//   Mutate temporary collection
// Return temporary collection
```
I look for a more concise, functional approach (i.e. using `map` or `reduce`).  I'll let you consider other approaches before I throw my suggestion in here.  😄